test/integration/user.js

```diff
@@ -3,57 +3,151 @@ import path from 'path';
 
 import chai from 'chai';
 
-import { Dropbox } from '../../index.js';
+import { Dropbox, DropboxAuth } from '../../index.js';
 import { DropboxResponse } from '../../src/response.js';
+import { DropboxResponseError } from '../../src/error.js';
 
-describe('User', () => {
-  const dbx = new Dropbox({ accessToken: process.env.DROPBOX_TOKEN });
+const appInfo = {
+  LEGACY: {
```

Comment:
Again with consistency, can we match names with the sdk? Also, can we be explicit about the fact these are user tokens in the name? i.e. `LEGACY_USER_DROPBOX_ACCESS_TOKEN`